public/data/AnuragK.json

```diff
@@ -0,0 +1,33 @@
+{
+  "name": "Anurag Kumar",
+  "type": "personal",
+  "bio": "Open Source Enthusiast!",
+  "avatar": "https://github.com/hfr58gb.png",
+  "links": [
+    {
+      "name": "Follow me on GitHub",
+      "url": "https://github.com/hfr58gb",
+      "icon": "github"
+    },
+    {
+      "name": "Follow me on Twitter",
+      "url": "https://twitter.com/AnuragS41695054",
+      "icon": "twitter"
+    },
+    {
+      "name": "Follow me on Hashnode",
+      "url": "https://anuragk24.hashnode.dev",
+      "icon": "hashnode"
+    },
+    {
+      "name": "Follow me on Stackoverflow",
```

Comment:
I recommend capitalizing the `o` here. What do you think?
```suggestion
      "name": "Follow me on StackOverflow",
```

P.S.: You may need to make a PR to add the StackOverflow icon. 🙂


__
[![sema-logo](https://app.semasoftware.com/img/sema-tray-logo.gif)](https://semasoftware.com/gh) &nbsp;**Summary:** :question: I have a question

Reply:
yeah, updated it